web/src/lib/components/asset-viewer/photo-viewer.svelte

```diff
@@ -7,14 +7,14 @@
   import { isWebCompatibleImage } from '$lib/utils/asset-utils';
   import { getBoundingBox } from '$lib/utils/people-utils';
   import { shouldIgnoreShortcut } from '$lib/utils/shortcut';
-  import { type AssetResponseDto } from '@immich/sdk';
   import { useZoomImageWheel } from '@zoom-image/svelte';
   import { onDestroy, onMount } from 'svelte';
   import { fade } from 'svelte/transition';
   import LoadingSpinner from '../shared-components/loading-spinner.svelte';
   import { NotificationType, notificationController } from '../shared-components/notification/notification';
 
-  export let asset: AssetResponseDto;
+  export let assetId: string;
+  export let originalPath: string;
```

Comment:
It might be better to leave `export let asset: AssetResponseDto` and just use reactive declarations in this component. That allows you to keep the export the same with changes like in #7596